export const getWords = async () => {
    try {
        console.log('getWords');
    } catch (error) {
        console.error('Error getting words:', error);
        throw error;
    }
};
